testgrid/config/config.yaml

```diff
@@ -2642,6 +2676,18 @@ dashboards:
     test_group_name: ci-kubernetes-verify-release-1.9
   - name: test-go-1.9
     test_group_name: ci-kubernetes-test-go-release-1.9
+  - name: bazel-build-1.9
+    test_group_name: periodic-kubernetes-bazel-build-1-9
+  - name: bazel-test-1.9
+    test_group_name: periodic-kubernetes-bazel-test-1-9
+  - name: gce-kubeadm-1.9
+    test_group_name: ci-kubernetes-e2e-kubeadm-gce-1-9
+  - name: gce-kubeadm-1.8-on-1.9
+    test_group_name: ci-kubernetes-e2e-kubeadm-gce-1-8-on-1-9
+  - name: periodic-gce-kubeadm-1.9
+    test_group_name: periodic-kubernetes-e2e-kubeadm-gce-1-9
+  - name: gce-kubeadm-upgrade-1.8-1.9
```

Comment:
Add these kubeadm jobs to `sig-cluster-lifecycle-all` as well?

Reply:
I think they're all in this dashboard now.